cli-deploy.js

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env node
-require("./command-runner").run(async () => {
```

Comment:
Now that this command runner isn't wrapping the body of this function, can you unindent the rest of it?